README.md

```diff
@@ -279,3 +279,11 @@ $ cosa init https://github.com/coreos/fedora-coreos-config.git
 $ cosa fetch && cosa build
 ```
 See this [Stack Overflow question](https://stackoverflow.com/questions/26028971/docker-container-ssl-certificates) for additional discussion.
+
+#### Overriding RPMs locally
+
+To override the RPM packages requested in `src/config/manifest.yaml`,
+drop local RPM packages into a directory `overrides/rpm`. This will
+generate a `coreos-assembler-local-overrides` repository during the
+next build where the overriding packages will be pulled from. Then, run
+`cosa build` to rebuild with the local overrides.
```

Comment:
I think it would be worth some example steps:

```
mkdir -p overrides/rpm
cp /path/to/my/rpm ./overrides/rpm
cosa build
```

Reply:
Thanks, added!